README.md

```diff
@@ -1 +1,18 @@
-# guesthouse
\ No newline at end of file
+# guesthouse
```

Comment:
why you hate capital letters mate ? :stuck_out_tongue_winking_eye:  :maple_leaf: